aclk/schema-wrappers/node_info.cc

```diff
@@ -4,9 +4,15 @@
 
 #include "schema_wrapper_utils.h"
 
+static int label_add_to_map(const char *name, const char *value, LABEL_SOURCE ls, void *data) {
```

Comment:
since we have same thing in `aclk/schema-wrappers/chart_stream.cc` should we move this function to `schema_wrapper_utils` file? This way we can have don't need to have duplicates

Reply:
@ktsaou, please, take a look in this suggestion.

Reply:
done